src/components/SearchField.tsx

```diff
@@ -53,6 +53,10 @@ const SearchField: React.FC = () => {
 
   const [genreToSearch, setGenreToSearch] = useState<string>('none');
```

Comment:
please, change type to Genres | null
https://github.com/oleksandrstarov/sigma_js_internship_2021/blob/487042caecb3276120fe3aa1b394ca6b3d631c45/src/components/SearchField.tsx#L8

I know this is not your part, but still

Reply:
Thanks for the help with it. Fixed.